roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_clusterpolicy.yml

```diff
@@ -67,3 +67,27 @@
 
   - name: Failing because the ClusterPolicy CR cannot be created
     fail: msg="Failed because the ClusterPolicy CR cannot be created"
+
+- name: Test if the ClusterPolicy has the 'validator' stanza
```

Comment:
If I'm reading this correctly, looking at the validator stanza is how you tell apart < v1.7.0 and >= v1.7.0?

Can you clarify that this is the purpose of this check by renaming the register from `clusterpolicy_has_validator` to `cluster_version_atleast_v1_7_0` or adding a comment that explains that?